std_msgs/string[] names
geometry_msgs/Pose2D[] vendors
